src/api/spec/components/sponsors_component_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe SponsorsComponent, type: :component do
+  let(:sponsor) do
+    {
+      'icon' => 'sponsor_suse',
+      'url' => '/foo',
+      'description' => 'Sponsor foo',
+      'name' => 'foo'
+    }
+  end
+  let(:config) { { 'sponsors' => [sponsor] } }
+
+  it do
```

Comment:
We should also test when there are no sponsors. The view component would then render nothing.